### ISSUE TITLE
Topics: Topic list table overlaps the popup with topic actions

### ISSUE THREAD

**Issue by Ppojin** [provectus/kafka-ui#3896]
### Issue submitter TODO list

- [X] I've looked up my issue in [FAQ](https://docs.kafka-ui.provectus.io/faq/common-problems)
- [X] I've searched for an already existing issues [here](https://github.com/provectus/kafka-ui/issues)
- [X] I've tried running `master`-labeled docker image and the issue still persists there
- [X] I'm running a supported version of the application which is listed [here](https://github.com/provectus/kafka-ui/blob/master/SECURITY.md)

### Describe the bug (actual behavior)

Table on topics tab, cannot use `Recreate Topic` or `Remove Topic` because button is not visible

### App version
[fdd9ad9](https://github.com/provectus/kafka-ui/commit/fdd9ad9)

### Docker-compose IAAC configs
```yaml
version: '3.9'
services:
  kafka-ui:
    image: provectuslabs/kafka-ui
    container_name: kafka-ui:master
    ports:
      - "9000:8080"
    environment:
      KAFKA_CLUSTERS_0_NAME: "localhost"
      KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS: "kafka1:9092,kafka2:9093,kafka3:9094"
      KAFKA_CLUSTERS_0_ZOOKEEPER: "zookeeper:2181"
      KAFKA_CLUSTERS_0_READONLY: "false"
```

### Steps to reproduce

1. Connect kafka cluster `READONLY=False`, that have just one topic
2. cluster -> topics -> click Dropdown Toggle

### Screenshots

![image](https://github.com/provectus/kafka-ui/assets/25076967/6a473108-5eb8-43fe-9734-71d266073856)

**Comment by Haarolean** [comment]
Hi, these buttons are not being hidden in any case, if there are no permissions or the cluster is r/o -- we disable them instead.

**Comment by Haarolean** [comment]
Considering that you have just one topic there, it seems that these buttons are hidden behind a topic list container, that'd be a bug indeed.

**Comment by Ppojin** [comment]
Hello @Haarolean, thank you for considering this issue.
It was occur by `style="position:relative"` on lower version of react-menu@3.3.0.
And it was fixed on [react-menu@3.3.0 src/__tests__/SSR.test.js](https://github.com/szhsin/react-menu/pull/553/files#diff-dbe48b7e1336f7b36a23522b891ca77fc0670da38b1f6410e3b7e64c00fc127d)

You can find on this [PR#3898](https://github.com/provectus/kafka-ui/pull/3898)
